pkg/skaffold/runner/build_deploy.go

```diff
@@ -49,18 +49,18 @@ func (r *SkaffoldRunner) Build(ctx context.Context, out io.Writer, artifacts []*
 		return nil, err
 	}
 
-	// In dry-run mode or with --digest-source  set to 'remote' or 'tag', we don't build anything, just return the tag for each artifact.
-	if r.runCtx.DryRun() || (r.runCtx.DigestSource() == remoteDigestSource) ||
-		(r.runCtx.DigestSource() == tagDigestSource) {
-		var bRes []build.Artifact
-		for _, artifact := range artifacts {
-			bRes = append(bRes, build.Artifact{
-				ImageName: artifact.ImageName,
-				Tag:       tags[artifact.ImageName],
-			})
-		}
-
-		return bRes, nil
+	// In dry-run mode or with --digest-source set to 'remote' or 'tag' in render, we don't build anything, just return the tag for each artifact.
+	switch {
+	case r.runCtx.DryRun():
+		color.Yellow.Fprintln(out, "Skipping image build since --dry-run=true")
```

Comment:
nit: I would change the wording from `image build` to `build phase` or something similar. When I read `Skipping image build ...` it sounds like it's skipping a single image rather than the whole phase

Reply:
yeah great point, updated